src/PythonClient/tests/quixstreams/test_dataframes/test_kafka/test_producer.py

```diff
@@ -0,0 +1,105 @@
+import confluent_kafka
+import pytest
+
+
+async def test_producer_start_close(producer):
+    async with producer:
+        ...
+
+
+class TestAsyncProducer:
+    async def test_producer_produce_non_blocking(self, producer, topic_factory):
+        topic_name, _ = await topic_factory()
+        async with producer:
+            await producer.produce(
+                topic=topic_name,
+                key="test",
+                value=b"test",
+            )
+
+    async def test_producer_produce_blocking_success(self, producer, topic_factory):
+        topic_name, _ = await topic_factory()
+        async with producer:
+            await producer.produce(
+                topic=topic_name, key="test", value=b"test", blocking=True
+            )
+
+    async def test_producer_produce_blocking_failure_raises_error(
```

Comment:
This test has failed for me twice in a row:

```       async with producer_factory(extra_config=extra_config) as producer:
>           with pytest.raises(confluent_kafka.KafkaException) as raised:
E           Failed: DID NOT RAISE <class 'cimpl.KafkaException'>

quixstreams/test_dataframes/test_kafka/test_producer.py:39: Failed
```

Reply:
Thanks for spotting that!
I fixed it in 599622de1a72c24a319f2e227814fb2cffb9f3b4